app/imports/api/user/StudentProfileCollection.ts

```diff
@@ -466,9 +466,9 @@ class StudentProfileCollection extends BaseProfileCollection {
    * @throws {Meteor.Error} If user is not defined.
    */
   public getProjectedICE(user: string) {
-    const studentID = Users.getID(user);
-    const courseDocs = CourseInstances.findNonRetired({ studentID });
-    const oppDocs = OpportunityInstances.findNonRetired({ studentID });
+    const userID = Users.getID(user);
+    const courseDocs = CourseInstances.findNonRetired({ userID });
```

Comment:
CourseInstances don't have a field userID. This is true about OpportunityInstances.